tests/RouteTest.php

```diff
@@ -420,6 +390,17 @@ public function testImmutability(): void
         $this->assertNotSame($route, $route->disableMiddleware(''));
     }
 
+    public function testBuiltMiddlewares(): void
+    {
+        $route = Route::get('')
+                      ->middleware(TestMiddleware1::class)
+                      ->action(static fn () => new Response(200));
+
+        $builtMiddlewares = $route->getData('builtMiddlewares');
```

Comment:
Due to the PR is going to raise the major version of package let's remove `getData` to regular getter?

Reply:
> Due to the PR is going to raise the major version of package let's remove `getData` to regular getter?

Rename `getData()`  to `get()` or create separate getters?

Reply:
The second option

Reply:
This is very non-comfortable when configre routes. IDE suggestions will get in the way in application, but getters in most cases need into packages only.